packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2336,7 +2336,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   void _onCursorColorTick() {
     renderEditable.cursorColor = widget.cursorColor.withOpacity(_cursorBlinkOpacityController.value);
-    _cursorVisibilityNotifier.value = widget.showCursor && _cursorBlinkOpacityController.value > 0;
+    _cursorVisibilityNotifier.value = widget.showCursor && _hasFocus && _cursorBlinkOpacityController.value > 0;
```

Comment:
I guess the caret was blinking when the field had a selection even when it wasn't focused.

Reply:
Any suggestions on process of recovering data with this issue